cargo-dist/tests/snapshots/akaikatana_basic.snap

```diff
@@ -1220,20 +1220,26 @@ class AkaikatanaRepack < Formula
   desc "The akaikatana-repack application"
   homepage "https://github.com/mistydemeo/akaikatana-repack"
   version "0.2.0"
-  if OS.mac?
-    if Hardware::CPU.arm?
-      url "https://github.com/mistydemeo/akaikatana-repack/releases/download/v0.2.0/akaikatana-repack-aarch64-apple-darwin.tar.xz"
-    end
-    if Hardware::CPU.intel?
-      url "https://github.com/mistydemeo/akaikatana-repack/releases/download/v0.2.0/akaikatana-repack-x86_64-apple-darwin.tar.xz"
-    end
-  end
-  if OS.linux?
-    if Hardware::CPU.intel?
-      url "https://github.com/mistydemeo/akaikatana-repack/releases/download/v0.2.0/akaikatana-repack-x86_64-unknown-linux-gnu.tar.xz"
-    end
-  end
+
```

Comment:
We need to be a little more careful about adding too much extra whitespace here - let's try to trim that back.